keras_cv/layers/object_detection/retina_net_label_encoder.py

```diff
@@ -36,6 +36,11 @@ class RetinaNetLabelEncoder(layers.Layer):
         background_class: (Optional) The class ID used for the background class.
             Defaults to -1.
         ignore_class: (Optional) The class ID used for the ignore class. Defaults to -2.
+        track_matched_boxes: (Optional) Whether or not to track the number of boxes that
```

Comment:
discussed offline -- let's just turn on this without impacting API

Reply:
(This is in the encoder - RetinaNet still doesn't have a way to disable)

Reply:
Updated